packages/x-date-pickers/src/AdapterLuxon/index.ts

```diff
@@ -33,12 +33,10 @@ const formatTokenMap: MuiFormatTokenMap = {
   LL: 'month',
   LLL: { sectionName: 'month', contentType: 'letter' },
   LLLL: { sectionName: 'month', contentType: 'letter' },
-  LLLLL: { sectionName: 'month', contentType: 'letter' },
```

Comment:
We can't correctly support those single letter month since they can represent several months (and are by definition impossible to parse).